.github/workflows/validate-taskfile.yml

```diff
@@ -0,0 +1,19 @@
+name: Valiate example Taskfile.yml
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  Validate-Example-Taskfile:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - run: |
+          curl -O https://json.schemastore.org/taskfile.json
```

Comment:
I'm surprised that this isn't pinned to a version as well.

Reply:
Actually, should we just commit this to our repo? `schemastore.org` isn't loading for me, I doubt this would change much, so perhaps save a tiny bit of time?

Reply:
I had it committed initially, but I thought this would be easier in the long run. Let's see how stable schemastore.org is and we can check it in if it's consistently wobbly